packages/lumx-react/src/components/comment-block/CommentBlock.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { mdiDotsHorizontal, mdiHeart, mdiReply } from '@lumx/icons';
+import { Button, CommentBlock, Emphasis, Size } from '@lumx/react';
+import { IconButton } from '@lumx/react/components/button/IconButton';
+import React from 'react';
+
+export default { title: 'LumX components/comment-block/CommentBlock' };
+
+export const WithHeaderActions = ({ theme }: any) => (
+    <CommentBlock
+        hasActions
+        hasChildren
+        actions={
+            <>
+                <Button key="button0" emphasis={Emphasis.low} size={Size.s} leftIcon={mdiHeart}>
+                    24 likes
+                </Button>
+                ,
```

Comment:
Is this coma a leftover of a transformation from array to fragment ? ^^

Reply:
Oops, yes. And actually I didn't want to do this transformation, I was just trying things out. :sweat_smile: I'll add the array back.